backend/.env

```diff
@@ -0,0 +1,8 @@
+POSTGRES_HOST=127.0.0.1
+POSTGRES_PORT=5432
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=mfasecretpassword
+POSTGRES_DATABASE=mfa_database
+PORT=3001
```

Comment:
Can we add this to the `.gitignore`? We can just send this in our slack channel and download it ourselves